crates/hir-def/src/macro_expansion_tests/builtin_fn_macro.rs

```diff
@@ -456,13 +456,13 @@ fn test_concat_expand() {
 #[rustc_builtin_macro]
 macro_rules! concat {}
 
-fn main() { concat!("fo", "o", 0, r#""bar""#, "\n", false, '"', '\0'); }
+fn main() { concat!("fo", "o", 0, r#""bar""#, "\n", false, '"', -4, '\0'); }
```

Comment:
```suggestion
fn main() { concat!("fo", "o", 0, r#""bar""#, "\n", false, '"', -4, - 4, '\0'); }
```
lets also throw in a space for good measure